### PR TITLE
Theme: Get input default styles working

### PR DESCRIPTION
**Pull request by ianrohde**
We needed to leverage `parts` to style the Input `field`: https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/input.ts#L3

qa_req 0

### CODE REVIEW

**Review by djmetzle — APPROVED** [review]
CR :inbox_tray:

File: chakraTheme/components/Input.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import cp from '@core-ds/primitives';
+
+const Input = {
```
Comment (rjmccluskey) [inline chakraTheme/components/Input.ts:3]: Is there an existing type you could use here? Typescript will infer the type when `Input` is used somewhere else. But within this file it's basically `any`.

**Comment by rjmccluskey** [comment]
CR 🍻
deploy_block 🍻 with a question about using a type

File: chakraTheme/components/Input.ts (hunk 1 above)
Comment (bhuminson) [reply chakraTheme/components/Input.ts:3]: I don't think so,  chakra doesn't type these https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/input.ts

**Comment by bhuminson** [comment]
un_deploy_block :dagger: 
i set QA reqs to 0 because this is just a prototype in the works

File: chakraTheme/components/Input.ts (hunk 1 above)
Comment (probablyian) [reply chakraTheme/components/Input.ts:3]: I spoke with Rich about this today and he had some good insight on ways to leverage types. I'd like to explore this again later as we get our themes more established.

**Pull request merged**